### DISCUSSION TITLE
Is it okay to rewrite entire project?

### DISCUSSION THREAD

**Discussion by seo-rii** [ANLAB-KAIST/KENSv3#211] [Project Q&A 4. Etc] [answered]
I've written enough code to solve PA2, but I ran into trouble expanding the code for PA3. Rewriting all the codes seems easier than modifying the current code, so is there any problem if I submit a completely different code to PA2?

**Comment by Pusnow** [comment, marked as answer]
It is okay to submit  a completely different code.
